doctest update metrics

nataliyah123 opened the pull request:
Fixes part of # 2066

bot-of-gabrieldemarmiesse commented:
@aakashkumarnain @marload 

You are owners of some files modified in this pull request.
Would you kindly review the changes whenever you have the time to?
Thank you very much.

WindQAQ commented on tensorflow_addons/metrics/cohens_kappa.py:
```diff
@@ -38,28 +38,30 @@ class CohenKappa(Metric):
+    >>> m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
+    >>> # m.update_state(actuals, preds)
+    >>> # print('Final result: ', m.result().numpy())
+    >>> # Final result: 0.61904764
```
Why these are commented out?

nataliyah123 replied:
please see the image below. 
I don't reckon what do you mean by 'all results from metric(y_true, y_pred) should be named result'. do you mean whenever a metric is directly called or do you mean metric.update_state(y,ypred), should be assigned to result?. if I assign update_state to result the print statement should be like
`print('Final result: ', result.result().numpy())` 
can you please check this [PR](https://github.com/tensorflow/addons/pull/2109). it has a similar issue as described below. I have not reverted back the changes 
![image](https://user-images.githubusercontent.com/5617255/92317247-38243600-f018-11ea-9647-42e3814fc4a6.png)

WindQAQ commented on tensorflow_addons/metrics/cohens_kappa.py:
```diff
@@ -38,28 +38,30 @@ class CohenKappa(Metric):
+    >>> m = tfa.metrics.CohenKappa(num_classes=5, sparse_labels=True)
+    >>> # m.update_state(actuals, preds, sample_weight=weights)
+    >>> # print('Final result: ', m.result().numpy())
+    >>> # Final result: 0.37209308
```
ditto

nataliyah123 replied:
it's not complete ditto the second example uses weights.

WindQAQ commented on tensorflow_addons/metrics/matthews_correlation_coefficient.py:
```diff
@@ -42,18 +42,18 @@ class MatthewsCorrelationCoefficient(tf.keras.metrics.Metric):
+    >>> preds = tf.constant([[1.0], [0.0], [1.0], [1.0]],
+    ... dtype=tf.float32)
+    >>> # Matthews correlation coefficient
+    >>> mcc = MatthewsCorrelationCoefficient(num_classes=1)
```
`tfa.metrics.MatthewsCorrelationCoefficient`?

nataliyah123 replied:
I had my doubt about this but since it was working completely fine. So, I left it as it is.

WindQAQ reviewed (changes requested):
Thanks for the contribution! Can we canonicalize the variable names? Like

- `actuals` -> `y_true`
- `preds` -> `y_pred`
- all metric instance should be named `metric`
- all results from `metric(y_true, y_pred)` should be named `result`.

nataliyah123 commented:
@WindQAQ can I change the pytest .ini file and remove DONT_ACCEPT_BLANKLINE because the output is not clearing the test

Harsh188 commented:
> @WindQAQ can I change the pytest .ini file and remove DONT_ACCEPT_BLANKLINE because the output is not clearing the test

Can you try adding `#doctest: -DONT_ACCEPT_BLANKLINE `

nataliyah123 commented:
working

nataliyah123 commented:
@googlebot I fixed it.

nataliyah123 commented:
@googlebot I fixed it.

nataliyah123 commented:
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
>  **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Faddons%2Fpull%2F2144) for more info**.
@googlebot I fixed it.

nataliyah123 commented:
> > We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> > In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> > **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Faddons%2Fpull%2F2144) for more info**.
> > @googlebot I fixed it.

@googlebot rescan

nataliyah123 commented:
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
>  **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Faddons%2Fpull%2F2144) for more info**.

@googlebot I fixed it.

nataliyah123 commented:
@googlebot I fixed it.

WindQAQ commented:
This is probably stale as #2171 exists. Let me close it but feel free to reopen it if I make the mistake.
